src/gui/PianoView.cpp

```diff
@@ -312,22 +409,23 @@ void PianoView::modelChanged()
  *
  *  \param _p The point that the mouse was pressed.
  */
-int PianoView::getKeyFromMouse( const QPoint & _p ) const
+int PianoView::getKeyFromMouse(const QPoint& _p) const
 {
 	int offset = _p.x() % PW_WHITE_KEY_WIDTH;
-	if( offset < 0 ) offset += PW_WHITE_KEY_WIDTH;
-	int key_num = ( _p.x() - offset) / PW_WHITE_KEY_WIDTH;
+	if (offset < 0)
```

Comment:
Instead of enforcing brackets and keeping the oneliner, we get an implicit body with indentation. Weird.

Reply:
OK, with ColumnLimit 120, the indentation is not touched anymore. However, it does not add the brackets yet. Do we want them in this case? At least our style guide says so for blocks *in general*. Any opinions?

Reply:
IMO we should keep explicit brackets unless we want to transition to something very bracket-less, which I'm unsure if C++ supports or not. Regardless that would be a really big change, probably controversial, and not something I'd push for.

Reply:
> 
> 
> IMO we should keep explicit brackets unless we want to transition to something very bracket-less, which I'm unsure if C++ supports or not. Regardless that would be a really big change, probably controversial, and not something I'd push for.

Keep explicit brackets, it saves heartache in the future.

Reply:
So, `clang-format` can't add curly brackets. However, it also cannot remove them if we add them. `clang-tidy` can do it, however, and I have added it already to clang-tidy (I just did not execute it yet on the code). Resolving this.